lib/src/puppetdb-cli.cc

```diff
@@ -116,6 +208,54 @@ size_t write_body(char *ptr, size_t size, size_t nmemb, void *userdata){
     return size * nmemb;
 }
 
+class JsonQueueWrapper {
```

Comment:
So this appears to be an adapter whose API is known by rapidjson, is that right? A name or comment or both to explain that would be good